File: app/src/main/java/br/edu/ifce/engcomp/francis/radarpolitico/models/Falta.java
package br.edu.ifce.engcomp.francis.radarpolitico.models;

/**
 * Created by Joamila on 24/03/2016.
 */
public class Falta {
    private String idDeputado;
    private String data;
    private String status;

    public Falta(String data, String status){
        this.data = data;
        this.status = status;
    }

    public String getIdDeputado() {
        return idDeputado;
    }

    public void setIdDeputado(String idDeputado) {
        this.idDeputado = idDeputado;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }




}

File: app/src/main/kotlin/br/edu/ifce/engcomp/francis/radarpolitico/models/Falta.kt
package br.edu.ifce.engcomp.francis.radarpolitico.models

import br.edu.ifce.engcomp.francis.radarpolitico.models.Deputado

/**
 * Created by Joamila on 24/03/2016.
 */
data class Falta(var data: String = "", var status: String = ""){
    var idDeputado: String? = null
}

